packages/data-validation/src/services/data-validator-registry.service.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { toDisposable } from '@univerjs/core';
+import { BehaviorSubject } from 'rxjs';
+import type { BaseDataValidator } from '../validators/base-data-validator';
+
+export enum DataValidatorRegistryScope {
+    SHEET = 'sheet',
+}
+
+/**
+ * Register data validator
+ */
+export class DataValidatorRegistryService {
+    private _validatorByScopes = new Map<string, Array<BaseDataValidator>>();
+
+    private _validatorMap = new Map<string, BaseDataValidator>();
+
+    private _validatorsChange$ = new BehaviorSubject<void>(undefined);
+
+    validatorsChange$ = this._validatorsChange$.asObservable();
```

Comment:
You could remove L33 so that the relationship between `_validatiorsChange$` `validatorsChange$` would be more clear.